README.md

```diff
@@ -24,7 +24,13 @@ $ npm install @finn-no/cdn-uploader -g
 
 Actual usage:
 ```sh-session
-$ cdn-uploader /tmp/cdn-assets -a test-app
+$ cdn-uploader /tmp/cdn-assets --cwd /tmp/cdn-assets -a test-app
```

Comment:
I would `--cwd` options would imply this should be written as:
`cdn-uploader ./ --cwd /tmp/cdn-assets -a test-app`